app/assets/javascripts/govuk_publishing_components/visual-regression.js

```diff
@@ -0,0 +1,56 @@
+window.GOVUK = window.GOVUK || {}
```

Comment:
We're polluting globals, this should be wrapped with an IIFE.

Reply:
@nickcolley What's the best way to test this tool if wrapping in an IIFE?

Reply:
If you check the accessibility-test file, we expose only the parts we need globally.

Reply:
(What's an IIFE?)

Reply:
https://stackoverflow.com/questions/8228281/what-is-the-function-construct-in-javascript